src/App.tsx

```diff
@@ -1,14 +1,82 @@
+/* eslint-disable react-hooks/rules-of-hooks */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { UserWarning } from './UserWarning';
+import { Loader } from './components/Loader';
+import { TodoFilter, FilterTypes } from './components/TodoFilter';
+import { Todos } from './components/Todos';
 
-const USER_ID = 0;
+import { client } from './utils/fetchClient';
+
+import { Todo } from './types/Todo';
+
+const USER_ID = '10682';
 
 export const App: React.FC = () => {
   if (!USER_ID) {
     return <UserWarning />;
   }
 
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [createTodo, setCreateTodo] = useState<string>('');
+  const [filter, setFilter] = useState(FilterTypes.All);
+  const [isLoader, setIsLoader] = useState<boolean>(true);
+
+  useEffect(() => {
+    setIsLoader(true);
+
+    client.get(USER_ID);
+  }, []);
+
+  const handleAddTodo = (
+    e: React.KeyboardEvent<HTMLElement>,
+  ) => {
+    if (createTodo.trim() !== '' && e.key === 'Enter') {
+      const newTodo: Todo = {
+        id: Date.now(),
+        userId: Date.now(),
+        title: createTodo,
+        completed: false,
+      };
+
+      setTodos((prevTodos) => [...prevTodos, newTodo]);
+      setCreateTodo('');
+    }
+  };
+
+  const handleCreateTodo = (
+    e: React.ChangeEvent<HTMLInputElement>,
+  ) => {
+    setCreateTodo(e.target.value);
+  };
+
+  const removeTodo = (todoId: number) => {
+    setTodos(prevTodos => prevTodos.filter((todo) => todo.id !== todoId));
+  };
+
+  const filteredTodos = filter === FilterTypes.All
+    ? todos
+    : todos.filter((todo) => {
+      if (filter === FilterTypes.Completed) {
+        return todo.completed;
+      }
+
+      return !todo.completed;
+    });
+
+  const filterTodos = (
+    e: any,
```

Comment:
don't use any, specify the event type (but actually you may get rid of it using links)